test/print-routes.test.js

```diff
@@ -22,6 +22,21 @@ test('should print routes', async t => {
   t.ok(spy.calledWithMatch('debug', '└── / (GET|POST)\n'))
 })
 
+test('should print routes via cli', t => {
+  t.plan(2)
+
+  const spy = sinon.spy()
+  const command = proxyquire('../print-routes', {
+    './log': spy
+  })
+  command.cli(['./examples/plugin.js'])
+
+  setTimeout(() => {
+    t.ok(spy.called)
+    t.ok(spy.calledWithMatch('debug', '└── / (GET|POST)\n'))
+  }, 5000)
```

Comment:
why is this needed? can you please use fake timers? Or maybe refactor to avoid needing this in the first place?

Reply:
The `cli` doesn't return a promise or any other indication that it has completely finished so I opted in to just wait a bit.
I'm not sure what would be the best approach here. I could make the `cli` return a promise but that's imo not a very good solution. We could also use `process on exit` though I'm not sure how `tap` works with that. I'm not sure how I can use fake timers to handle that though.

Also, as I mentioned in a PR comment we could just avoid this test.

Reply:
Let's make the cli return a promise instead.

Reply:
Done